source/installation/install-software.rst

```diff
@@ -16,6 +16,12 @@ software requirements:
    This tutorial is run from the perspective of an account that has
    :command:`sudo` access but is not root.
 
+..  tip::
```

Comment:
I can't tell if this should be a `tip` or `warning`.

Reply:
I'm thinking `warning` myself, as it isn't conveying a useful bit of info but something that seems pretty necessary to know. I know that can also make it look scary, but I'd rather the reader noticed it than didn't.

Reply:
```suggestion
..  warning::
```